### PR TITLE
docs: translate build_and_run.md

### PR DESCRIPTION
**Pull request by wu-hanqing**
### What is changed and how it works?

What's Changed: translate build_and_run.md 

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: docs/cn/build_and_run.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ sudo yum install bazel git gcc-c++ make
```
Comment (SeanHai) [inline docs/cn/build_and_run.md:67]: add [English Version] doc link

File: docs/en/build_and_run.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
+
```
Comment (SeanHai) [inline docs/en/build_and_run.md:1]: add [Chinese Version] doc link

Comment (SeanHai) [inline docs/en/build_and_run.md:1]: The English docs are name as xxx_en.md now, so build_and_run_en.md is better

File: docs/en/build_and_run.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+# Build compilation environment
+
+**Note:**
+1. If you just want to experience the deployment and basic functions of CURVE, **you do not need to compile CURVE**, please refer to [Single-machine deployment](deploy.md).
```
Comment (SeanHai) [inline docs/en/build_and_run.md:5]: The "Single-machine deployment" doc link maybe (deploy_en.md)

File: docs/en/build_and_run.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+2. This document is only used to help you build the curve code compilation environment, which is convenient for you to participate in the development, debugging and packaging test of curve.
```
Comment (SeanHai) [inline docs/en/build_and_run.md:6]: "curve" should be "Curve" or "CURVE" be consistent

File: docs/en/build_and_run.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+
+Method 1: Pull the docker image from the docker hub image library (recommended)
+
+```bash
+$ docker pull opencurve/curvebuild:centos8
+```
+
+Method 2: Build docker image manually
+
+Use the dockerfile in the project directory to build. The command is as follows:
```
Comment (SeanHai) [inline docs/en/build_and_run.md:20]: change dockerfile to Dockerfile

File: docs/en/build_and_run.md [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+### Compile in docker image
+
+```bash
+$ docker run -it opencurve/curvebuild:centos8 /bin/bash
+$ cd <workspace>
+$ git clone https://github.com/opencurve/curve.git
+$ bash mk-tar.sh
+```
+
+For the installation and deployment process based on the tar package, please refer to: [Cluster Deployment](deploy.md).
```
Comment (SeanHai) [inline docs/en/build_and_run.md:38]: deploy.md to deploy_en.md

File: docs/en/build_and_run.md [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+
+### One-click compilation and packaging
+
+Curve provides a one-click compilation script, [mk-tar.sh](https://github.com/opencurve/curve/blob/master/mk-tar.sh) generates all the required tar binary packages, the command is as follows:
+
+```
+$ bash ./mk-tar.sh
+```
+
+For the installation and deployment process based on the tar package, please refer to: [Cluster Deployment](deploy.md)
```
Comment (SeanHai) [inline docs/en/build_and_run.md:74]: deploy.md to deploy_en.md

File: docs/en/build_and_run.md (hunk 2 above)
Comment (wu-hanqing) [reply docs/en/build_and_run.md]: fix

Comment (wu-hanqing) [reply docs/en/build_and_run.md]: fix

File: docs/en/build_and_run.md (hunk 3 above)
Comment (wu-hanqing) [reply docs/en/build_and_run.md]: fix, but about now the deploy_en.md is not accessable.

File: docs/en/build_and_run.md (hunk 4 above)
Comment (wu-hanqing) [reply docs/en/build_and_run.md]: fix

File: docs/en/build_and_run.md (hunk 5 above)
Comment (wu-hanqing) [reply docs/en/build_and_run.md]: fix

File: docs/en/build_and_run.md (hunk 6 above)
Comment (wu-hanqing) [reply docs/en/build_and_run.md]: fix

File: docs/en/build_and_run.md (hunk 7 above)
Comment (wu-hanqing) [reply docs/en/build_and_run.md]: fix

**Review by SeanHai — APPROVED** [review]

**Review by ilixiaocui — APPROVED** [review]

**Pull request merged**